internal/admin/server.go

```diff
@@ -0,0 +1,27 @@
+package admin
+
+import (
+	"net/http"
+	"net/http/pprof"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const DefaultAdminAddr = ":6060"
+
+// DefaultAdminServeMux is the default ServeMux to be used for admin servers in packages like httpbp, thriftbp, etc.
+// DefaultAdminServeMux configures the following routes:
+//    metrics       - serve /metrics for prometheus
+//    profiling     - serve /debug/pprof for profiling, ref: https://pkg.go.dev/net/http/pprof
+var DefaultAdminServeMux = http.NewServeMux()
```

Comment:
honestly just `Mux` is probably fine, though `ServeMux` would be fine by me as well.  HTTP calls it `DefaultServeMux` but in our case it's not the default, it's the only, and we don't have other kinds of muxes.

Reply:
we don't have any other kinds of muxes now, but could in the future.  I don't feel strongly either way tho.